CHANGELOG/CHANGELOG-3.7.md

```diff
@@ -24,7 +24,7 @@ Previous change logs can be found at [CHANGELOG-3.6](https://github.com/etcd-io/
 
 ### Dependencies
 
-- Compile binaries using [go 1.24.6](https://github.com/etcd-io/etcd/pull/20460)
+- Compile binaries with [Go 1.25](https://go.dev/doc/devel/release#go1.25.minor).
```

Comment:
Eventually we will need to specify the patch version when we release 3.7.0, but it's fine now